moviepy/audio/io/readers.py

```diff
@@ -179,9 +179,9 @@ def get_frame(self, tt):
             # Check that the requested time is in the valid range
             if not in_time.any():
                 raise IOError(
-                    "Error in file %s, " % (self.filename)
-                    + "Accessing time t=%.02f-%.02f seconds, " % (tt[0], tt[-1])
-                    + "with clip duration=%f seconds, " % self.duration
+                    f"Error in file {self.filename}, "
+                    + f"Accessing time t={tt[0]:.02f}-{tt[-1]:.02f} seconds, "
+                    + f"with clip duration={self.duration:f} seconds, "
```

Comment:
you could avoid the `+` and as the concatenation will be implicit 
https://docs.python.org/3/reference/lexical_analysis.html#string-literal-concatenation